src/main/topLevelPages/myDashboard/DashboardContext.tsx

```diff
@@ -17,14 +20,17 @@ export const DashboardProvider = ({ children }: { children: ReactNode }) => {
   const cachedView = getCachedView();
   const isActivityEnabled = cachedView === DashboardViews.ACTIVITY;
   const [activityEnabled, setEnabled] = useState<boolean>(isActivityEnabled);
+  const [openedDialog, setOpenedDialog] = useState<DashboardDialog | undefined>(undefined);
```

Comment:
No need of explicit `undefined` in the type and the value:
`const [openedDialog, setOpenedDialog] = useState<DashboardDialog>();` is absolutely the same and it's shorter.